docs/.pages

```diff
@@ -0,0 +1,7 @@
+nav:
+  - Welcome: README.md
```

Comment:
When I run the site locally (via `make run`) I don't see the welcome page, it gives a 404

Reply:
This is an item that needs to be discussed.  I actually wanted to handle it in a separate issue but now is a good time as any.

Imo, the current README.md needs to be renamed to welcome.md.  That renders the page when browsing directly to `/` and `..../welcome/`.  The README.md at the top level of the repo needs to be a real README containing who/what/when/where/how to operate and develop the site.  If your're good with this I can make that update.